Project 4/SPARQL-JV.md

```diff
@@ -0,0 +1,526 @@
+**The SPARQL Library of Buffalo**
+
+[Codewars](https://www.codewars.com/dashboard) is a website designed to facilitate algorithmic training for various programming languages. Users supply problem statements and others provide coding solutions to those problems. For example, you might find a problem for Python such as: 
```

Comment:
Dear @jonathanvajda,

The new changes look good! The kata 7s look on par and the rest of the changes that were suggested are a solid improvement from the last iteration. 

Well done! 

Thanks for inviting me to review your project!

Best, 
Jaron